### DISCUSSION TITLE
Python roadmap

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#39] [General]
![image](https://user-images.githubusercontent.com/1768891/135586237-de035e72-980f-49ae-9d17-ec2a3d51a986.jpeg)

Source: https://www.linkedin.com/in/mahmoud-essam-822719149

**Comment by enoch-lightning** [comment]
全部是英文，感覺頗複雜。因為老師說python裡面凡事皆物件(object)，所以這樣的心智圖(concept map)有特別與物件(object)的概念相關嗎？
